CI failing due to coveralls issue

```
=WARNING REPORT==== 19-Sep-2021::08:56:03.260747 ===
Description: "Authenticity is not established by certificate path validation"
     Reason: "Option {verify, verify_peer} and cacertfile/cacerts is missing"

Failed sending coverdata to coveralls, 405: "<html>\r\n<head><title>405 Not Allowed</title></head>\r\n<body>\r\n<center><h1>405 Not Allowed</h1></center>\r\n<hr><center>nginx</center>\r\n</body>\r\n</html>\r\n"
Error: Process completed with exit code 1.
```

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -69,7 +69,7 @@ jobs:
     - name: Create Cover Reports
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-      run: rebar3 do cover
+      run: rebar3 do cover, coveralls send
     - name: Produce Documentation
       run: rebar3 edoc
     - name: Publish Documentation

--- a/README.md
+++ b/README.md
@@ -3,7 +3,7 @@
 ![erlang_ls](images/erlang-ls-logo-small.png?raw=true "Erlang LS")
 
 ![Build](https://github.com/erlang-ls/erlang_ls/workflows/Build/badge.svg)
-[![Coverage Status](https://coveralls.io/repos/github/erlang-ls/erlang_ls/badge.svg?branch=master)](https://coveralls.io/github/erlang-ls/erlang_ls?branch=master)
+[![Coverage Status](https://coveralls.io/repos/github/erlang-ls/erlang_ls/badge.svg?branch=main)](https://coveralls.io/github/erlang-ls/erlang_ls?branch=main)
 
 An Erlang server implementing Microsoft's Language Server Protocol 3.15.
 

